desktop-gui/src/main/java/guiApp/GuiApp.java

```diff
@@ -0,0 +1,50 @@
+package guiApp;
+
+import configs.GuiConfig;
+import controllers.GuiController;
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.support.AbstractApplicationContext;
+import org.springframework.stereotype.Component;
+import ui.Ui;
+
+import javax.annotation.PreDestroy;
+
+@Component
+public class GuiApp extends Application implements Ui {
+    public void start(final Stage primaryStage) throws Exception {
+        final AbstractApplicationContext ctx = new AnnotationConfigApplicationContext(GuiConfig.class);
```

Comment:
Beans contain in Spring context of GUI don't found beans from context of core. It is means that we don't use core services.
I think that you should use [context hierarchy](http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/context/support/GenericApplicationContext.html#GenericApplicationContext-org.springframework.context.ApplicationContext-).